packages/react/src/components/Attachments/AttachmentWindow.js

```diff
@@ -124,13 +128,24 @@ function AttachmentWindow() {
                   Cancel
                 </button>
 
-                <button
-                  type="button"
-                  onClick={submit}
-                  className={styles.attachment_window_submit_button}
-                >
-                  Send
-                </button>
+                {!isPending && (
```

Comment:
Instead of code duplication, can't we use something like this ? 

```
<button
  type="button"
  onClick={submit}
  className={styles.attachment_window_submit_button}
  disabled={isPending}
>
  {isPending ? 'Sending...' : 'Send'}
</button>

```

Reply:
Also it would be great if you add this css to show that button is grayed out !

```
.attachment_window_submit_button[disabled] {
  background-color: rgb(168, 176, 179);
}

```